ci-operator/config/openshift-priv/router/openshift-priv-router-main.yaml

```diff
@@ -107,7 +107,7 @@ tests:
         echo "************ baremetalds e2e router conf command ************"
 
         read -d '#' INCL << EOF
-        [sig-network] Services should provide secure master service
+        [sig-network] Services should provide secure main service
```

Comment:
This looks like a mistake.

Reply:
Good catch and thank you @Miciah

The change is pushed.